src/js/control-bar/live-display.js

```diff
@@ -42,7 +42,7 @@ class LiveDisplay extends Component {
 
     this.contentEl_ = Dom.createEl('div', {
       className: 'vjs-live-display',
-      innerHTML: `<span class="vjs-control-text">${this.localize('Stream Type')}</span>${this.localize('LIVE')}`
+      innerHTML: `<span class="vjs-control-text">${this.localize('Stream Type')}&nbsp;</span>${this.localize('LIVE')}`
```

Comment:
we should use the unicode representation (`\u00a0`) for xhtml safety https://github.com/videojs/video.js/pull/4825